cmd/deploy/deploy.go

```diff
@@ -363,7 +363,7 @@ func (dc *Command) Run(ctx context.Context, deployOptions *Options) error {
 		return err
 	}
 
-	if dc.IsRemote || dc.RunningInInstaller {
+	if dc.RunningInInstaller {
```

Comment:
The whole if can be probably be removed, as the installer is now passing variables to the command using the `--var` flag, but left it here because it might imply other changes, and in the case of the installer is not a big deal, because when the deployment is executed from the UI (or through a preview environment) the variables are stored in the configmap. But when the deploy is executed directly from the CLI, variables are not yet stored in the configmap.

So, leaving that for future evaluation